jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -109,7 +110,7 @@ def test_parse_numpy_array_3d(self, imviz_helper, manual_loop):
         for i in range(n_slices):
             data = imviz_helper.app.data_collection[i]
             comp = data.get_component('DATA')
-            assert data.label == f'my_slices_{i}'
+            assert data.label == (f'my_slices ({i})' if i > 0 else 'my_slices')
```

Comment:
I don't know how to feel about this. I don't like the new labeling behavior wrt loading each slice of cube as an image, but we also have Cubeviz for cube, so maybe not a deal breaker...

Reply:
This PR doesn't actually introduce that behavior, right?  It only changes `my_slices_i` to `my_slices (i)` (and excludes the `(i)` for the first slice to be consistent with other duplicate labels)?

Reply:
When you load a cube as a series images, not having number for the first frame seems... wrong.

Reply:
In line 103:
```
for i in range(n_slices):
    imviz_helper.load_data(arr[i, :, :], data_label=f'my_slices_{i}', do_link=False)
```
would fix that since all data labels would be unique.